### PR TITLE
misc(publish): include the report bundle in npm package

### PR DESCRIPTION
**Pull request by paulirish**
to support folks like web.dev using our report renderer we need to include the new renderer API in our published assets

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -56,6 +56,7 @@ results.html
 dist/**
 !dist/report/standalone.js
 !dist/report/flow.js
+!dist/report/bundle.umd.js
```
Comment (connorjclark) [inline .npmignore:59]: Can we limit ourselves to the esm version?

**Comment by connorjclark** [comment]
What's wrong with a client using `lighthouse/report/renderer/report-renderer.js` ?

**Comment by paulirish** [comment]
> We could even provide an `exports` field in package.json so `lighthouse/renderer` would import `report-renderer.js`

looks good. tried it out with a local web.dev (thank you `build-pack`) and we're good.

esm, too. ;)

File: package.json [hunk 2]
```diff
@@ -3,6 +3,10 @@
   "version": "8.6.0",
   "description": "Automated auditing, performance metrics, and best practices for the web.",
   "main": "./lighthouse-core/index.js",
+  "exports": {
+    ".": "./lighthouse-core/index.js",
+    "./renderer": "./dist/report/bundle.esm.js"
+  },
```
Comment (connorjclark) [inline package.json:9]: When exports is defined, it means all exports are not allowed unless listed here. See the output from build-devtools:

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lighthouse-core/lib/i18n/i18n' is not defined by "exports" in /Users/runner/work/lighthouse/lighthouse/lighthouse/node_modules/lighthouse/package.json
```

Adding this line should fix it:

```
"./lighthouse-core/*": "./lighthouse-core/*"
```

**Comment by paulirish** [comment]
@connorjclark can we merge as is? i certainly don't want to block v9 release with a "lets lock down our node exports" exercise. We've already found that it's not trivial with just one of our many downstream customers. :(

**Review by connorjclark — APPROVED** [review]

**Pull request merged**